game-app/game-core/src/main/java/games/strategy/triplea/ui/TechnologyPanel.java

```diff
@@ -0,0 +1,288 @@
+package games.strategy.triplea.ui;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.data.events.GameDataChangeListener;
+import games.strategy.engine.stats.IStat;
+import games.strategy.triplea.Constants;
+import games.strategy.triplea.delegate.TechAdvance;
+import games.strategy.triplea.delegate.TechTracker;
+import java.awt.Component;
+import java.awt.GridLayout;
+import java.awt.Image;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import javax.swing.ImageIcon;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+import javax.swing.table.AbstractTableModel;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableCellRenderer;
+import javax.swing.table.TableColumn;
+import org.jetbrains.annotations.NotNull;
+
+class TechnologyPanel extends JPanel implements GameDataChangeListener {
+  private static final long serialVersionUID = 4340684166664492498L;
```

Comment:
Must be unique, but is duplicate to constant from StatPanel.java
Is there anything that really has to be stored/loaded on game save/load.
My understanding would be that the game load would trigger the constructor and rebuild the panel with the game data that has been loaded. Hence, nothing from this UI part needs to be saved into the game save file or read from it as all would run through the game data.